tests/appsec/iast/test_iast.py

```diff
@@ -21,10 +21,13 @@ class Test_Iast(BaseTestCase):
 
     if context.library == "nodejs":
         EXPECTED_LOCATION = "/usr/app/app.js"
+        WEAK_CIPHER_ALGORITHM = "des-ede-cbc"
```

Comment:
I think this is from a different PR?

Reply:
yes, this PR is for testing at this moment, I have to rebase the branch